src/Apps/Artist/Routes/Consign/Components/ArtistConsignHeader.tsx

```diff
@@ -46,3 +68,122 @@ export const ArtistConsignHeader: React.FC<ArtistConsignHeaderProps> = props =>
     </SectionContainer>
   )
 }
+
+/**
+ * Displays a random set of images for a given consignment artist.
+ *
+ * Some notes about implementation:
+ *
+ * 1) Due to responsiveness and SSR we need to dynamically scale the width and the
+ *    height of the image based on overall width of the container.
+ * 2) For this we use our `<ResponsiveImage>` component.
+ * 3) However, this component achieves responsivity by forcing padding within a
+ *    container and then applying a "fit" to a background image.
+ * 4) Therefore, conventional borders cannot be applied, and a CSS filter is
+ *    required. This is based upon pixel data, and border faked.
+ * 5) The pixel data comes from a second image, sitting below the first, with
+ *    its brightness turned to -100.
+ * 5) It's not perfect, and the layout needs to also account for variable
+ *    aspect ratios.
+ */
+const HeaderImages = (props: {
+  artworksByInternalID: Consign_artworksByInternalID
+}) => {
+  const getRandomImages = () => {
+    const [left, right] = sampleSize(props.artworksByInternalID, 2)
+
+    // Layout requires that we only use vertically oriented images
+    const widthExceedsHeight = ({ image }) => {
+      return image.resized.width > image.resized.height
+    }
+
+    if (widthExceedsHeight(left)) {
+      return getRandomImages()
+    }
+    if (widthExceedsHeight(right)) {
+      return getRandomImages()
```

Comment:
We're doing some recursion here, but the array size is small. I'm not worried.

Reply:
If I'm understanding correctly, it will recursively call itself until it chooses two images randomly that are both vertically oriented. Is it possible that we could have an artist who has no vertically oriented images (or only one), in which case this would never actually find two usable images?

I also think iterating through a randomly sorted collection of the images until two good ones are found would be easier to understand and safer than using recursion.

Reply:
I can add an iteration guard, but I think for the 5 or so lines here, recursion is more readable and much simpler, so gonna stick with this.

Reply:
I've added an iteration cap of 100 while also try / catching the entire function for further safety should any other problems occur. If caught, the images don't render.